decidim-bulletin_board-js/src/client/client.js

```diff
@@ -61,4 +61,16 @@ export class Client {
   processTallyStep(params) {
     return this.apiClient.processTallyStep(params);
   }
+
+  /**
+   * Query PendingMessages for a given messageId
+   *
+   * @param {Object} params - An object that include the following options.
+   *  - {String} messageId - The messageId
+   * @returns {Promise<Object>} - The status of a pendingMessage.
```

Comment:
It returns the whole `pendingMessage` now 😄

Reply:
😅